RELEASE/scripts/autoscend/quests/level_13.ash

```diff
@@ -1671,6 +1254,176 @@ boolean L13_towerNSTower()
 	return false;
 }
 
+boolean L13_towerNSTowerSkin()
+{
+	if(!contains_text(visit_url("place.php?whichplace=nstower"), "ns_05_monster1"))
+	{
+		return false;
+	}
+	auto_log_info("Time to fight the Wall of Skins!", "blue");
+	if (get_property("auto_towerBreak").to_lower_case() == "wall of skin" || get_property("auto_towerBreak").to_lower_case() == "wallofskin" || get_property("auto_towerBreak").to_lower_case() == "skin" || get_property("auto_towerBreak").to_lower_case() == "level 1")
+	{
+		abort("auto_towerBreak set to abort here.");
+	}
+	if (item_amount($item[Beehive]) > 0)
+	{
+		return autoAdvBypass("place.php?whichplace=nstower&action=ns_05_monster1", $location[Tower Level 1]);
+	}
+	// Can we kill the tower without a beehive?
+	beehiveConsider(true);
+	if(get_property("auto_getBeehive").to_boolean())
+	{
+		return false;
+	}
+	
+	int damage = 2; // base attack damage plus TT attack skill (kneebutt, headbutt)
+	
+	boolean fam_set = false;
+	foreach fam in $familiars[shorter-order cook, mu, imitation crab] // crab is evergreen, buy one
+	{
+		if (have_familiar(fam) && auto_is_valid(fam))
+		{
+			handleFamiliar(fam);
+			use_familiar(fam);
+			damage += 4; // worst one is crab, at 4.
```

Comment:
may as well use the actual value (fam == $familiar[imitation crab] ? 4 : 5)

Reply:
Done

Reply:
apparently shorter order cook is 6